docs/docs/getting-started/web.mdx

```diff
@@ -174,7 +181,7 @@ export default function App() {
   return (
     <WithSkiaWeb
       opts={{ locateFile: (file) => `https://cdn.jsdelivr.net/npm/canvaskit-wasm@${version}/bin/full/${file}` }}
-      getComponent={() => import("./MySkiaComponent")}
+      getComponent={() => import("./MySkiaComponent")} // use require(...) if you're using the metro bundler
   );
 }
 ```
```

Comment:
Looks like there is something wrong here?

Reply:
It _looks like_ it indeed, but it's having the same odd behavior in the [current `web.mdx` page](https://github.com/Shopify/react-native-skia/blob/main/docs/docs/getting-started/web.mdx):
<img width="1541" alt="CleanShot 2023-11-30 at 16 48 44@2x" src="https://github.com/Shopify/react-native-skia/assets/2856923/a09bc0ec-4566-4593-9e07-436ecc16d90b">

I've really just added a comment `//...` and nothing more (like line 136), so that should work fine.